lua/nvim-treesitter/parsers.lua

```diff
@@ -1908,6 +1908,15 @@ list.scss = {
   maintainers = { "@elianiva" },
 }
 
+list.sflog = {
+  install_info = {
+    url = "https://github.com/aheber/tree-sitter-sfapex",
+    files = { "src/parser.c" },
+    location = "sflog",
+  },
+  maintainers = { "@aheber" },
```

Comment:
Did @aheber volunteer to maintain _the queries_ in _this repo_?

Reply:
@clason  yes, I'm in closely contact with him, and agreed upon this with him.

Reply:
Yes, this can have my name on it.

Reply:
Well, it's less about whose name is on it but who I can ping about issues and PRs (and breaking changes) and expect a response...

Reply:
This lang parser sits in the repo owned by @aheber, but I'm the one who would actively respond to pings about it.
Shall I update the name to my name @clason ?

Reply:
Yes (and same for the other parsers in that repo, if @aheber is also not maintaining those queries here).

Reply:
As long as he's available for questions about them, that's OK then. (We've had very bad experiences with maintainers who basically stopped responding as soon as the PR was merged; I want to avoid that going forward.)

Reply:
So, yes, if _you_ are responsible for the queries, then _you_ should be the maintainer. (Since the parser is not developed in this repo, this is not very relevant.)

Reply:
This won't be the case on our parsers, even if we don't work in the Salesforce ecosystem in the future, we will make sure somebody taking them over. 

And I have just updated apex, sosl, soql, sflog maintainers to my name.

Reply:
As I said, I'm more concerned about the _queries_ ;) (But that is good to hear; we've also had problems with abandoned parsers which required adaptation to upstream tree-sitter changes to keep working...)

Reply:
I'm curious, do you get users requesting query updates in this repo? I think they should ask the listed maintainers.

I'm asking because I'd be so happy to see requests in our parsers. It means there are real people using and caring about them 😸

Reply:
Yes, since the queries are editor-specific and curated to be consistent (as far as possible) across languages. For issues that require parser changes, we push to report them at the parser repo and close them here.

Reply:
Got it, all clear. Updating the query knowledge is something I never find the excuse to take action, hopefully one day driven by the user requests ;).

Reply:
I'm happy for @xixiaofinland to be the listed maintainer here. I'll gladly help with any query stuff or parser stuff that comes up and I've been responsive here in the past when my parser change has broken these queries, preventing updates.
I do plan to keep that going but have seen enough languishing projects to understand the concern.

Is there a way to provide a secondary or backup maintainer?

Reply:
(Nothing of that was personal, mind you; just explaining why I am pressing on that point.)

And the maintainers field is a table for a reason ;)

Reply:
Thanks, @clason!

@xixiaofinland, would you be interested in having us both as listed maintainers? 

Changing the relevant sections to: 
`maintainers = { "@xixiaofinland", "@aheber" }`

That would help get any problems resolved more quickly and add more long-term stability.

Reply:
@aheber let's do this, I added both of our names :)